### DISCUSSION TITLE
Video capture using SK Remoting, Possible?

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#643] [Q&A]
I'm using SK Remoting and wonder if anybody has been able to do video capture on the remote device?

**Comment by maluoi** [comment]
I have done this before, but there are a few caveats to it! Since the rendering is happening remotely, it seems like it doesn't do the third eye render stuff. This means there's some misalignment with the real world, hands especially. It also seemed to me like the footage was a lot shakier? But that may have been my imagination.

For reference, here's a video capture of SK using HolographicRemoting:
https://twitter.com/koujaku/status/1559689459171414017

**Comment by scrooksjr** [comment]
I would look into WebRTC... to capture video on a remote device. I built an application for AVNIK Defense Solutions Inc. for a similar purpose. Good luck!
